src/legacy/core_plugins/vis_type_vega/public/vega_fn.ts

```diff
@@ -19,16 +19,35 @@
 
 import { get } from 'lodash';
 import { i18n } from '@kbn/i18n';
+
+import { ExpressionFunction, KibanaContext, Render } from '../../interpreter/types';
+import { VegaVisualizationDependencies } from './plugin';
 import { createVegaRequestHandler } from './vega_request_handler';
 
-export const createVegaFn = (dependencies) => ({
-  name: 'vega',
+const name = 'vega';
+type Context = KibanaContext | null;
```

Comment:
@lukeelmers talked to @alexwizp 
Were unclear about the meaning of the `Context` type, in the scope of `ExpressionFunction`?

Reply:
@lizozom @alexwizp  `Context` is the output of one function that gets passed to the next. For example, in this expression:

```
esaggs aggConfig="blah blah" | vega spec="hi"
```

The value returned by the `esaggs` function and piped into the `vega` function is called "Context". 

Each expression function defines which context types it accepts as `context`... https://github.com/elastic/kibana/blob/f9aca228213653941d00afd94e6709a408b4cc8c/src/legacy/core_plugins/vis_type_vega/public/vega_fn.ts#L49-L51

...And which context type it returns (`Return` interface) as `type`: https://github.com/elastic/kibana/blob/f9aca228213653941d00afd94e6709a408b4cc8c/src/legacy/core_plugins/vis_type_vega/public/vega_fn.ts#L48